xrpl/wallet/main.py

```diff
@@ -16,23 +17,53 @@ class Wallet:
     details.
     """
 
+    @property
+    def address(self: Wallet) -> str:
+        """
+        Alias for wallet.classic_address.
+
+        Returns:
+            The address that publicly identifies this wallet, as a base58 string.
+        """
+        return self.classic_address
+
+    @address.setter
+    def address(self: Wallet, value: str) -> None:
```

Comment:
Why should this be set-able?

Reply:
If someone wants to do `wallet.address = "something"`, we would need this setter

Reply:
We should discourage that, especially now that there's regular key support.

Reply:
I think the reverse should be true - you shouldn't be able to set either `wallet.classic_address` or `wallet.address` after initialization.